website/static/js/contribManager.js

```diff
@@ -462,11 +423,39 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
 
 function ContribManager(selector, contributors, adminContributors, user, isRegistration, table, adminTable) {
     var self = this;
+    //shouter allows communication between ContribManager and ContribRemover, in particular which contributor needs to
+    // be removed is passed to ContribRemover
+    var contribShouter = new ko.subscribable();
```

Comment:
I didn't see any error messages.  I assumed it was a timing issue or the inability of a signal event to travel between views.